pkg/testcase/certificates.go

```diff
@@ -0,0 +1,122 @@
+package testcase
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+)
+
+func errCheck(error error, message string, ignore bool) {
+	if error != nil {
+		shared.LogLevel("error", message)
+		if !ignore {
+			Expect(error).NotTo(HaveOccurred())
+		}
+
+	}
+}
+
+// Rotate certificate for etcd only and cp only nodes
+func certRotate(product string, ip string) {
+	// Stop service on server
+	_, stopError := shared.StopService(product, ip, "server")
+	errCheck(stopError, fmt.Sprintf("Error stopping %s service", product), false)
+	time.Sleep(time.Duration(10) * time.Second) // Adding 10 second sleep between steps
+	// Rotate certificate
+	_, rotateError := shared.CertRotate(product, ip)
+	errCheck(rotateError, fmt.Sprintf("Error running certificate rotate for %s service", product), false)
+	time.Sleep(time.Duration(10) * time.Second) // Adding 10 second sleep between steps
+	// start service on server
+	_, startError := shared.StartService(product, ip, "server")
+	errCheck(startError, fmt.Sprintf("Error starting %s service", product), true)
+	time.Sleep(time.Duration(10) * time.Second) // Adding 10 second sleep between steps
+	// Retry Stop start of service. (Inconsistent failure to start service above,
+	// can be fixed hopefully with a restart)
+	if startError != nil {
+		_, retryError := shared.StopStartService(product, ip, "server")
+		errCheck(retryError, fmt.Sprintf("ERROR: Retrying Stop/Start %s service", product), false)
```

Comment:
error messages dont start with capital should be  "error...."

Reply:
fixed

Reply:
not fixed 

```
		shared.ReturnLogError(fmt.Sprintf("Error running certificate rotate for %s service on %s", product, ip))
	}

```

Reply:
same for all error messages on this method